amd_openvx_extensions/amd_media/decoder.cpp

```diff
@@ -741,6 +754,11 @@ void CLoomIoMediaDecoder::DecodeLoop(int mediaIndex)
                 return;
             }
             int status = avcodec_receive_frame(videoCodecContext[mediaIndex], frame);
+#if ENABLE_PERF_MEASURE           
```

Comment:
remove this code (line 757-761) and put it after this [line](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/master/amd_openvx_extensions/amd_media/decoder.cpp#L757)